scripts/build_script.sh

```diff
@@ -48,13 +48,18 @@ if [ -n "$INTEGRATIONS_CORE_REF" ]; then
     export INTEGRATIONS_CORE_VERSION="$INTEGRATIONS_CORE_REF"
 fi
 
+INVOKE_TASK="omnibus.build"
+if ! inv --list | grep -F "${INVOKE_TASK}"; then
```

Comment:
I think you want to add the `-q` flag to grep in this case
```suggestion
if ! inv --list | grep -qF "$INVOKE_TASK"; then
```

Reply:
Ah indeed, good point!